### DISCUSSION TITLE
Εξεταστική Σεπτεμβρίου και παρουσιάσεις προόδου

### DISCUSSION THREAD

**Discussion by siotasandreas** [courses-ionio/hci#1775] [Announcements]
Καλησπέρα σας,
 
 Είμαι επί πτυχίω φοιτητής και θα ήθελα να ρωτήσω αν στην τωρινή εξεταστική του Σεπτεμβρίου θα υπάρξει παρουσίαση των εργασιών. Αν ναι, υπάρχει δυνατότητα να παρουσιάσω διαδικτυακά λόγω εργασιακών υποχρεώσεων;

Ευχαριστώ εκ των προτέρων

**Comment by epidrome** [comment]
Σας παρακαλώ να μην ρωτάμε-απαντάμε τα ίδια, 

διαβάστε προσεκτικά τις οδηγίες
https://courses-ionio.github.io/help/classroom/

καθώς και ίδιες ερωτήσεις-απαντήσεις στο ιστορικό
https://github.com/courses-ionio/help/discussions/514
